src/plugins/dashless/index.ts

```diff
@@ -0,0 +1,48 @@
+import { ReactNative } from "src/metro/common/libraries/index.ts";
+import { after } from "src/patcher/index.ts";
+import { definePlugin } from "#plugin-context";
+import { Dev } from "@data/constants";
+
+const { View } = ReactNative;
+
+const traverseAndModify = (node: any): any => {

```

Comment:
Wouldn't this replace every string node instead of only "changing dashes in text channel names to spaces"?